### PR TITLE
After documentation

### PR DESCRIPTION
**Pull request by DominikPinsel**
PR for the documentation commits of the SSI library. As this is mainly documentation, maybe it can be merge regardless of its size.

### CODE REVIEW

**Comment by DominikPinsel** [comment]
@SebastianBezold Again pretty big, but mostly documentation. I hope we can merge it :)

**Comment by carslen** [comment]
Please resolve merge conflicts.

File: docs/Feature-Create-Did.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+* CreateDidOptions - Marker interface. Implementations hold properties required to create a new DID of the respective **DidMethod**.
+* DidFactoryRegistry - *MAY* be used to register **DidFactory** implementations for multiple **DidMethod**s
+* DidWebMethod - Example implementation of **DidMethod** for method *did:web*.
+* CreateDidWebOptions - Example implementation of **CreateDidOptions** for method *did:web*.
+* Did - Value class representing a DID. *MAY* refer to a **DidDocument**
+* DidDocument - Value class representing a DID document.
+
+
+
+
```
Comment (florianrusch-zf) [inline docs/Feature-Create-Did.md:38]:
```suggestion

```

**Review by florianrusch-zf — CHANGES_REQUESTED** [review]
I would recommend running a Markdown linter over the changes!
The EDC already has a Github workflow for this, which could also be included in this repo: https://github.com/eclipse-tractusx/tractusx-edc/blob/5a7ff105a080fa83199dad87c8b7fe5a062cc632/.github/workflows/verify.yaml#L79-L89

If I'm correct, the linter and the respective config should be already aligned with our github page repo: https://github.com/eclipse-tractusx/eclipse-tractusx.github.io

**Pull request closed without merging**